src/main/resources/data/minebound/patchouli_books/minebound_guide_book/en_us/entries/crafting_intro/crafting_intro.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Intro To Crafting",
+  "icon": "minebound:energized_focus",
+  "category": "minebound:introduction",
+  "pages": [
+    {
+      "type": "patchouli:text",
+      "text": "All recipes for Minebound are compatible with JEI and can be found there. To access our new recipes you'll need a Crafting bench, Armor Forge, and Alloy Furnace. The progression of our crafting goes from Energized Iron -> Illuminant Steel -> Gilded Diamond -> Myrial Diamond -> Myrial Effigy -> Myrial Suit -> Myrial Synergy -> Myrial Singularity"
```

Comment:
Each Myrial Armor corresponds to a level of ore; I'm not sure how this displays, but it probably shouldn't look like you need myrial diamond to craft myrial effigy